src/pages/p2p/components/_availability.js

```diff
@@ -102,7 +106,24 @@ const Line = styled.div`
     }
 `
 
+const Left = styled.div`
+    width: 78%;
+`
+
+const Right = styled.div`
+    width: 22%;
+`
+
+const query = graphql`
+    query {
+        qr_code: file(relativePath: { eq: "p2p_playstore.png" }) {
```

Comment:
can we move it to a folder named p2p for example?

Reply:
Sure